cmd/root.go

```diff
@@ -32,6 +32,7 @@ var bSilent bool
 var allocUnderRepair bool
 
 var walletJSON string
+var storageSdk sdk.StorageSdkSchema
```

Comment:
I will suggest to renamed `storageSdk` with `sdk`, and import `sdk` as storageSdk. so all changes can be removed on all commands.

Reply:
Noted. I'll change that.

Reply:
It'll create conflict with another sdk import.